fretboard.el

```diff
@@ -31,20 +32,20 @@
 ;; To highlight semitones, prefix the key with # (yes, prefix!).
 
 ;;; Code:
+(require 'symbol-overlay)
 
 (defgroup fretboard nil
-  "A guitar fretboard visualization.")
+  "A guitar fretboard visualizer."
+  :group 'fretboard)
```

Comment:
Is such a recursive customization group a common thing?

Reply:
It won't hurt anything. Technically you should add it to one of the default groups.
Applications is probably the appropriate choice.